test/joke_test.rb

```diff
@@ -1,17 +1,132 @@
 require 'minitest/autorun'
 require 'minitest/pride'
 require './lib/joke'
+require './lib/user'
+require './lib/open_mic'
 
 class JokeTest < Minitest::Test
```

Comment:
We would love to see this test broken up into separate files for the User and OpenMic class testing!